src/main/kotlin/com/dansplugins/factionsystem/command/faction/claim/MfFactionClaimCommand.kt

```diff
@@ -19,11 +19,37 @@ import java.util.logging.Level.SEVERE
 import kotlin.math.floor
 
 class MfFactionClaimCommand(private val plugin: MedievalFactions) : CommandExecutor, TabCompleter {
+    private val factionClaimAutoCommand = MfFactionClaimAutoCommand(plugin)
     private val factionClaimFillCommand = MfFactionClaimFillCommand(plugin)
+    private val factionClaimCheckCommand = MfFactionClaimCheckCommand(plugin)
+
+    private val autoAliases = listOf("auto", plugin.language["CmdFactionClaimAuto"])
+    private val fillAliases = listOf("fill", plugin.language["CmdFactionClaimFill"])
+    private val checkAliases = listOf("check", plugin.language["CmdFactionClaimCheck"])
+    // private val circleAliases = listOf("circle", plugin.language["CmdFactionCheckClaim"])
+    // private val squareAliases = listOf("square", plugin.language["CmdFactionCheckClaim"])
+    // private val rectangleAliases = listOf("rectangle", plugin.language["CmdFactionCheckClaim"])
+
+    private val subcommands = autoAliases + fillAliases + checkAliases
 
     private val decimalFormat = DecimalFormat("0", DecimalFormatSymbols.getInstance(plugin.language.locale))
 
     override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<out String>): Boolean {
+        return when (args.firstOrNull()?.lowercase()) {
+            null -> MfFactionClaimCommand(sender, command, label, args.drop(1).toTypedArray())
```

Comment:
This can go in the else block - it provides a little more feedback to the user than the current else